website/docs/api/cli.md

```diff
@@ -1163,6 +1164,38 @@ $ python -m spacy evaluate [model] [data_path] [--output] [--code] [--gold-prepr
 | `--help`, `-h`                            | Show help message and available arguments. ~~bool (flag)~~                                                                                                                           |
 | **CREATES**                               | Training results and optional metrics and visualizations.                                                                                                                            |
 
+## benchmark {#benchmark new="3.5"}
+
+The `spacy benchmark` CLI includes commands for benchmarking the accuracy and
+speed of your spaCy pipelines.
+
+### accuracy {#benchmark-accuracy new="3.5" tag="command"}
+
+This subcommand is an alias for [`spacy evaluate`](#evaluate).
```

Comment:
Maybe we should also have a note over at the `evaluate` documentation? Realistically we won't be deprecating `evaluate` anytime soon, but it's always good to warn long up-front.

Reply:
It's a bit weird anyway to have all the documentation under `evaluate` if `benchmark speed` is supposed to be the replacement. So I moved the docs to `benchmark speed` and changed the docs for `evaluate` to state that it is superseded and that it is an alias for compatibility.